frontend/src/component/changeRequest/ChangeRequestOverview/ChangeRequestTimeline/ChangeRequestTimeline.tsx

```diff
@@ -44,28 +52,53 @@ export const determineColor = (
     displayStageIndex: number,
 ) => {
     if (changeRequestState === 'Cancelled') return 'grey';
+
     if (changeRequestState === 'Rejected')
         return displayStage === 'Rejected' ? 'error' : 'success';
     if (
         changeRequestStateIndex !== -1 &&
-        changeRequestStateIndex >= displayStageIndex
```

Comment:
What did this solve?

Reply:
We want a `warning` color when change request is in `Scheduled` state where as all the other states have a `success` color.  This is why the `=` was handled separately